src/Routes.tsx

```diff
@@ -0,0 +1,29 @@
+import { Navigate, Route, Routes as RouterRoutes } from 'react-router-dom';
+
+import { LogInLayout, PageLayout, SingInLayout } from 'layouts';
+import { CheckEmail, ForgotPassword, Home, LogIn, NewPassword, SingIn } from 'pages';
+
+const Routes = () => {
+  return (
+    <RouterRoutes>
+      <Route path='/' element={<PageLayout />}>
+        <Route index element={<Home />} />
+      </Route>
+
+      <Route path='/login' element={<LogInLayout />}>
+        <Route index element={<LogIn />} />
+        <Route path='ForgotPassword' element={<ForgotPassword />} />
```

Comment:
1. I think you should rename it using the snake case `forgot-password`
2. Please move forgot password to it is own main route

Reply:
or you can use map to iterate through object